tests/common.rs

```diff
@@ -0,0 +1,54 @@
+extern crate pretty_assertions;
+
+extern crate failure;
+extern crate inferno;
+
+use failure::Fail;
+use inferno::flamegraph;
+use pretty_assertions::assert_eq;
+use std::error::Error;
+use std::fs::{self, File};
+use std::io::{BufRead, BufReader, Cursor};
+
+pub fn generate_and_check(
+    input_file: &str,
+    expected_result_file: &str,
+    options: flamegraph::Options,
+) -> Result<(), Box<dyn Error>> {
```

Comment:
I think we actually probably _want_ to use `unwrap` in tests, because it means we'll get a better read on exactly where the error occurred. Is there a particular reason you wanted to use `?` instead?

Reply:
I found the message is not very useful for the context(git was cloned without submodules):
```
---- flamegraph_nameattr stdout ----
thread 'flamegraph_nameattr' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/libcore/result.rs:997:5
```
Sure stack race can help, but real context is required for tests to understand the issue without stacktrace. I considered that `Result` in tests should help, but looks I was wrong.

In this case `expect` is more relevant than `unwrap`.  It should provide an additional context.

BTW, I deleted this commit.